packages/shared/src/components/Sidebar.tsx

```diff
@@ -25,9 +29,10 @@ interface SidebarMenuItem {
   title: string;
   path?: string;
   action?: () => unknown;
+  notif?: ReactElement;
```

Comment:
Would suggest to rename to `notification` just to keep everything written out in the codebase.

Reply:
I agree or we can probably use `alert` and reserve the `notification` for in-app notifs. Thanks for the thought!